components/discord_webhook/actions/send-message-with-file/send-message-with-file.js

```diff
@@ -0,0 +1,73 @@
+const discordWebhook = require("../../discord_webhook.app.js");
+var request = require("request");
```

Comment:
[`request` is deprecated](https://www.npmjs.com/package/request). Could you use `axios`, instead?

Reply:
Updated to use `axios`